include/VecCore/Backend/Implementation.h

```diff
@@ -284,6 +285,21 @@ VECCORE_FORCE_INLINE VECCORE_ATT_HOST_DEVICE constexpr Bool_s EarlyReturnAllowed
 #endif
 }
 
+// An extension of EarlyReturnAllowed(), and new name is a reminder that it can be used for other purposes as well
+template <typename T, size_t simdSize = 1>
+VECCORE_FORCE_INLINE
+VECCORE_ATT_HOST_DEVICE
+constexpr Bool_s SIMDsizeUpTo()
```

Comment:
`SIMDSizeAtMost` is a better name than `SIMDSizeUpTo`. What I dislike is the fact that somehow CUDA gets mixed up in ways that make it return false for something it should not, like the case for double. One thing is the hardware capability, another thing is an algorithm. I don't want to mix up the concepts. If someone later uses VecCore and expects SIMDSizeAtMost(1) be true for scalar, and the function returns false somehow, what am I to say? If in VecGeom you want this behavior, that's ok, but VecCore cannot and will not make such assumptions. Whatever function we add has to do what it says, and the name needs to be clear. Some suggestions are: `VectorSizeAtMost`, `VectorSizeAtLeast`, instead of SIMD, because SIMD is a bit too specific, and we already have a `VectorSize` function, not `SIMDSize`.